README.md

```diff
@@ -0,0 +1,135 @@
+<!--
+Example command:
+
+```sh
+ gem install rails
+```
+ -->
+
+### Setup
+
+Clone this repository to your desired folder: 
+
+
+ Commands:
+
+```sh
+  cd hello-microverse-project
+  https://github.com/rodrigopenayo98/hello-microverse-project.git
+```
+
+### Install
+
+Install this project with:
+
+<!--
+Example command:
+
+```sh
+  cd my-project
+  gem install
+```
+--->
+
+### Usage
+
+To run the project, execute the following command:
+
+<!--
+Example command:
+
+```sh
+  rails server
+```
+--->
+
+### Run tests
+
+To run tests, run the following command:
+
+<!--
+Example command:
+
+```sh
+  bin/rails test test/models/article_test.rb
+```
+--->
+
+### Deployment
+
+You can deploy this project using:
+
+<!--
+Example:
+
+```sh
+
+```
+ -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+
+
+👤 **Rodrigo Penayo**
+
+- GitHub: [@rodrigopenayo98](https://github.com/rodrigopenayo98)
+- Twitter: [@rodrigopenayo98](https://twitter.com/rodrigopenayo98)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/rodrigo-penayo-391226158/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+- [ ] **[new_feature_1]**
+- [ ] **[new_feature_2]**
+- [ ] **[new_feature_3]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+
+If you like this project give a star
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+
+I would like to thank Microverse
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./MIT.md) licensed.
+
+_
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- You have done wonderfully well by adding a Readme file that allows users or visitors to have a wholesome understanding of your project, also for others who would like to use it :clap: :clap: 
- [ ] However, just as the previous code reviewer indicated, there's a template required by Microverse to make your repo look really professional. I understand that you have used some parts of it, but every section or content of the template is actually compulsory except for `Microverse logo` (which must be removed or replaced) and the `FAQ` section which is OPTIONAL.
  - [ ] These sections are missing in your ReadMe:
    - [ ] _Table of Contents_
    - [ ] _Project description_
    - [ ] _Built with_
    - [ ] _Tech stack_
    - [ ] _Key features_
    - [ ] _Getting started - prerequisites_
    
- The whole essence of ReadMe is to share personalzed or custom contents regarding the project by sharing descriptive information with users, and you have done really well by removing some unnecessary comments (template guides) in the template used.
- [ ] Kindly add the missing contents as stated above, and please :pray: do not forget to personalise the content. Make it your own. Recruiters can be impressed with your good documentation skills, and going forward, you won't have issues with ReadMe in your future projects. :100: :wink: 

_For your reference, here is the [ReadMe template](https://github.com/microverseinc/readme-template)_